modules/ROOT/pages/policy-openid-connect.adoc

```diff
@@ -111,7 +111,7 @@ You can obtain the client ID of the OAuth2 token by using the following DataWeav
 
 == Communicating with the Token Validation Endpoint Using a Proxy
 
-You can enable the OpenID Connect OAuth 2.0 Token Enforcement policy to use the gateway proxy settings by specify the following property when starting the Mule runtime engine: 
+You can enable the OpenID Connect Access Token Enforcement policy to use the gateway proxy settings by specify the following property when starting the Mule runtime engine: 
```

Comment:
```suggestion
You can enable the OpenID Connect Access Token Enforcement policy to use the gateway proxy settings by specifying the following property when starting Mule runtime engine: 
```
I know it is wacky to omit the article, but despite it's capitalization, "Mule runtime engine" is the complete product name...sigh....

Reply:
will fix